packages/legend-query-builder/src/components/explorer/QueryBuilderPropertySearchPanel.tsx

```diff
@@ -354,23 +402,44 @@ export const QueryBuilderPropertySearchPanel = observer(
                       propertySearchPanelState.searchText,
                   },
                 )}
-                onChange={changePropertyName}
+                onChange={onSearchPropertyTextChange}
                 value={propertySearchPanelState.searchText}
-                placeholder="Search for a property"
+                placeholder={`Search for a property (min ${QUERY_BUILDER_PROPERTY_SEARCH_TEXT_MIN_LENGTH.toString()} chars)`}
               />
               {!propertySearchPanelState.searchText ? (
-                <div className="query-builder-property-search-panel__input__search__icon">
-                  <SearchIcon />
-                </div>
-              ) : (
                 <>
-                  <div className="query-builder-property-search-panel__input__search__count">
-                    {propertySearchPanelState.filteredPropertyNodes.length}
+                  <button
+                    className="query-builder-property-search-panel__input__cog__icon"
+                    tabIndex={-1}
+                    onClick={toggleSearchPanelTip}
+                    title="View tips for searching"
```

Comment:
Remember that these are not really `tips`, they are `Show advanced search configuration`